Apps/Src/ReadTritium.c

```diff
@@ -17,14 +17,7 @@ void Task_ReadTritium(void* p_arg) {
 		if(status == SUCCESS) {
 
 			OSQPost(&CANBus_MsgQ, (void *) &buf, sizeof(buf), OS_OPT_POST_FIFO, &err);
-			

```

Comment:
Why is the error checking removed? If you don't care about the contents of the function, remove it.

Reply:
Ok